Broadcast request gets send, but not played/broadcasted to the speakers

I was able to get the container up & running, but when I send request via Insomnia/CURL/Home Asisstant I get the following lines in the logs of the container:
![image](https://user-images.githubusercontent.com/62308520/170834189-a2ff2683-a1ee-486a-a729-12710515b1ed.png)

But the speakers (Google Nest Audio's) do not broadcast the message.
If I look at the Google Console Platform I see that the request do get send but still they aren't played:
![image](https://user-images.githubusercontent.com/62308520/170834336-20badecc-fb05-47d2-b9eb-c924ed8d9e2b.png)


The client_secret is made for my main google account which is also the owner of my google home, so it should have all the rights/persmissions it needs.

My question is did I forget to configure some things? 

- In your setup do you have to do specific settings to tell it which speakers it could broadcast to? (I have Google Nest speakers which I can broadcast to with Google Assistant via my phone/car)
- In the OAuth consent screen do you need to publish the APP? I did not and have it set tot external user type with myself (main google account email address) as a test user.

Sadly still nothing, still the same results.
Then I was trying to use your new environment variable langue, but got some errors:
```
google-assistant-broadcast    | [ERROR] Error while broadcasting: Error: 3 INVALID_ARGUMENT: Invalid 'dialog_state_in': bad language_code.
google-assistant-broadcast    |     at Object.callErrorFromStatus (/usr/src/app/node_modules/@grpc/grpc-js/build/src/call.js:31:26)
google-assistant-broadcast    |     at Object.onReceiveStatus (/usr/src/app/node_modules/@grpc/grpc-js/build/src/client.js:391:49)
google-assistant-broadcast    |     at Object.onReceiveStatus (/usr/src/app/node_modules/@grpc/grpc-js/build/src/client-interceptors.js:328:181)
google-assistant-broadcast    |     at /usr/src/app/node_modules/@grpc/grpc-js/build/src/call-stream.js:187:78
google-assistant-broadcast    |     at process.processTicksAndRejections (node:internal/process/task_queues:77:11) {
google-assistant-broadcast    |   code: 3,
google-assistant-broadcast    |   details: "Invalid 'dialog_state_in': bad language_code.",
google-assistant-broadcast    |   metadata: Metadata {
google-assistant-broadcast    |     internalRepr: Map(1) { 'grpc-server-stats-bin' => [Array] },
google-assistant-broadcast    |     options: {}
google-assistant-broadcast    |   }
google-assistant-broadcast    | }
```

With  `LANGUAGE='en-GB'` of `LANGUAGE='nl-BE'`, weird because those languages/locales should be supported or is that the wrong variable?

--- a/Dockerfile
+++ b/Dockerfile
@@ -1,4 +1,4 @@
-FROM node:17.8.0-alpine3.15
+FROM node:18.2.0-alpine3.15
 
 LABEL maintainer="Ismar Slomic <ismar@slomic.no>"
 

--- a/README.md
+++ b/README.md
@@ -61,7 +61,6 @@ services:
       - "8085:8085"
     volumes:
       - /home/pi/config:/usr/src/config
-
 ```
 
 ```bash
@@ -128,3 +127,34 @@ curl -X POST http://localhost:8085/broadcast \
  -d '{"message":"Hello world!!"}' \
  -H "Content-Type: application/json"
 ```
+
+## Known issues
+#### 1. Broadcast request gets send, but not played/broadcast to the speakers
+I was able to get the container up & running, but when I send broadcast request via 
+Postman/CURL/Home Asisstant no message is played on my speakers and I get the following lines in 
+the logs (more details about this issue at [#5](https://github.com/ismarslomic/google-assistant-broadcast/issues/5)):
+```(bash)
+Sending message: Broadcast Hello world!
+[OK] Conversation Response:  empty
+[OK] Conversation Completed
+```
+This issue is related to the language settings we set in the container, when using the 
+Google Assistant SDK (default is `en-GB`). It's hard to say exactly what the issue is, but for me 
+it helped to switch from `en-US` to `en-GB`. It might also be necessary to play with the language 
+settings in the Google Assistant app for the same Google account.
+
+You can try changing the language settings by setting the environment variable `LANGUAGE` to one of the 
+[supported languages codes](https://developers.google.com/assistant/sdk/reference/rpc/languages).
+
+Add `-e LANGUAGE=<language_code>` option if running with `docker run` command or `environment` if you 
+use `docker-compose`:
+
+```yaml
+version: '3.8'
+services:
+  google-assistant-broadcast:
+    ...
+    environment:
+      - LANGUAGE=<language_code>
+```
+

--- a/assistant.js
+++ b/assistant.js
@@ -12,7 +12,7 @@ const config = {
     savedTokensPath: process.env.TOKEN || "/usr/src/config/tokens.json"
   },
   conversation: {
-    lang: 'en-US', // defaults to en-US, but try other ones, it's fun!
+    lang: process.env.LANGUAGE || 'en-GB', // defaults to en-GB, but try other ones, it's fun!
     showDebugInfo: true, // default is false, bug good for testing AoG things,
     isNew: true,
   },

--- a/package.json
+++ b/package.json
@@ -22,7 +22,7 @@
   },
   "homepage": "https://github.com/ismarslomic/google-assistant-broadcast#readme",
   "dependencies": {
-    "express": "^4.17.3",
+    "express": "^4.18.1",
     "google-assistant": "^0.7.0"
   }
 }

